### ISSUE TITLE
A11y_Windows Template Studio_ Services Your project Details_Screenreader: Screenreader does not announce Alert Indication status after enter numbering in Edit field.

### ISSUE THREAD

**Issue by harshgangrade111** [microsoft/WindowsTemplateStudio#3894]
[“Check out Accessibility Insights! ](https://accessibilityinsights.io/)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags:** #A11yTCS; ;#GH_WindowsTemplateStudio_Oct2020;#Windows Template Studio;#DesktopApp;#Win10;#WCAG3.3.3;#NonBenchmark;#FTP;#STP;#NVDA;#Narrator;#A11yMAS;#A11ySev2;

### **Environment Details:**
Application Name: Windows Template Studio (WinTS)
Templates version: 3.8.20273.1
Wizard version: 3.8.20273.1Date: 2020-06-17T21:13:20.174Z
Microsoft Visual Studio Enterprise 2019 Preview
Version 16.8.0 Preview 3.1
Windows Version: Windows10
NVDA Version: 2019.3.1

### **Repro Steps:**

1.	Open  Visual Studio 2019 
2.	Click on File from menu and then select new and then project
3.	Select the "Windows template studio(WPF.Net Core)".Click on next
4.	"Configure your new project" screen will appear. Click on create button.
5.	New WPF App screen will open. Navigate to "Services" in the left navigation menu  and click on it.
6.	"Add Services "screen will appear. Navigate to "your Project Details" with the help of keyboard tab key.
7.	Start Narrator and move to Pages Edit field with the help of keyboard Tab key then type any numbering keys.
8.	Narrator does not announce alert indication.

### **Actual:**
Screen reader does not announce Alert indication status after enter numbering in Edit field.

**Same issue is also observed when we delete any  **list item** from Features section present in this screen.**

**Same issue is also observed in Scenario "Your Project Details"**.

**Same issue is also observed with following Scenario:**

- Select The Template You Want To Add

**Same issue is also observed with NVDA.**

### **Expected:**
Screen Reader should be announce Alert status which is occurring after edit any text in Edit field.

### **User Impact:**
Screen user users will not be able to get information about alert message if screen reader does not announce anything.

**Comment by sibille** [comment]
Verified in dev-nightly version:
Wizard version: 0.22.20310.1

You can install the dev-nightly version from our nightly feed (https://www.myget.org/F/windows-template-studio-nightly/vsix/).
For instructions on how install WTS from the dev-nightly feed in Visual Studio see https://github.com/microsoft/WindowsTemplateStudio/blob/dev/docs/getting-started-extension.md#nightly--pre-release-instructions

**Comment by harshgangrade111** [comment]
Issue does not repro in the latest dev-nightly version. Hence closing This bug.
